### DISCUSSION TITLE
How to duplicate a site (WordPress)

### DISCUSSION THREAD

**Discussion by babrees** [leokhoa/laragon#559] [Q&A] [answered]
I have a template site in Laragon that I want to simply duplicate to set up a new site

I tried the clone feature, but when I visit the site it shows the Laragan Getting Started page.  
I've been to phpMyAdmin and that is definitely a clone. I changed the url to the new cloned site. 
If I go to wp-admin it returns a 404.
I've checked the config file, which is correct.

Is Clone is the right way to do this?

**Comment by ho30hero** [comment, marked as answer]
Hello
One Click Clone (for me for WordPress installation) is really needed. 
I have to turn back to WP Studio local host for this reason.
Please add it to Laragon. 
kind Regards

**Reply by ho30hero** [reply]
Please :)

**Reply by leokhoa** [reply]
@ho30hero : Laragon has a Clone feature.

**Comment by BruTru** [comment]
Do you try copy/paste under Windows ?

**Reply by AWqxKAWERbXo** [reply]
This will copy the files, Not the database

**Reply by BruTru** [reply]
Use export/import in phpMyAdmin

**Comment by BruTru** [comment]
You have the line siteurl into the table wp_options. If you clone your website,, you'll have to change the value
